src/server/server_family.cc

```diff
@@ -1478,7 +1480,8 @@ void ServerFamily::Info(CmdArgList args, ConnectionContext* cntx) {
     append("listpack_blobs", total.listpack_blob_cnt);
     append("listpack_bytes", total.listpack_bytes);
     append("small_string_bytes", m.small_string_bytes);
-
+    append("maxmemory", max_memory_limit);
```

Comment:
why it was added? we have this at line 1466

Reply:
thanks for catching this, these two lines came from resolving a previous merge conflict... I didn't notice in the head it has been moved there.